docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - predictive_capacity_network
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
-      interval: 120s
+      interval: 60s
```

Comment:
At some point, we should ask ourselves why have different values that the one by default (see [documentation](https://docs.docker.com/reference/dockerfile/#healthcheck)). I'm not sure we're doing the most efficient things here.

Reply:
I don't have strong opinions about the value, I just noticed the service is the slowest to start.